docs/pages/_app.js

```diff
@@ -153,22 +153,24 @@ function AppWrapper(props) {
     let productIdentifier;
     let pages = [];
 
-    if (productId === 'toolpad') {
+    if (productCategoryId === 'toolpad') {
+      if(productId === 'core') {
       productIdentifier = {
         metadata: '',
-        name: 'Toolpad',
+        name: 'Toolpad Core',
         versions: [{ text: `v0.0.1`, current: true }],
       };
       pages = toolpadPages;
     }
-    if (productId === 'toolpad-studio') {
```

Comment:
Haven't started a full review yet, but we should probably keep `toopad-studio`, this being a global (across the company) product id for Algolia